lib/upgrader.ml

```diff
@@ -656,15 +656,14 @@ let make ~prefix ~old_file ~old_file_version ~new_file ~new_file_version =
     in
     let impl_header =
       [%string
-        {|module $old_version = $old_version_t
-module $new_version = $new_version_t
+        {|
```

Comment:
why do you remove the module aliases in the implementation?
I don't get how it works without it.